docs/application_profile.md

```diff
@@ -70,6 +71,7 @@ The DLME Prototype Application Profile a subset of the [Europeana Data Model](ht
 | cho_is_part_of   | `dct:isPartOf`    |             | 0...n       | literal/ref | A related resource in which the described resource is physically or logically included. | |
 | cho_language     | `dc:language`     | M+          | 0...n       | literal     | Language for the resource. | Mandatory for text in Europeana; determine if BCP47 language tags are feasible |
 | cho_medium       | `dct:medium`      |             | 0...n       | literal     | The material or physical carrier of the resource. | |
+| cho_periodo_period | | R | 0...n | literal/ref | URI for the periodo period matching the cho_temporal string value | |
```

Comment:
The property column should match to a specific RDF property; I can take a look and make a recommendation later.

Reply:
@jacobthill Is the intent to have a Periodo URI associated with the value in `cho_temporal`?

Reply:
If that's the case, I think maybe `edm:hasMet` is the right property here?

Reply:
Yes, that is the intent. If the string in the cho_temporal field matches a string in the periodo data, I fetch the periodo uri/s. `edm:hasMet` might be right though I'm not sure. Let me see if that makes sense to Wayne and Marcin.